src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -53,14 +54,23 @@
     private static ApiUser apiUser;
     private static OtpUser otpUser;
 
+    /**
+     * Whether tests for this class should run. End to End must be enabled and Auth must NOT be disabled. This should be
+     * evaluated after the middleware application starts up (to ensure default disableAuth value has been applied from
+     * config).
+     */
+    private static boolean testsShouldRun() {
+        return isEndToEnd() && !isAuthDisabled();
```

Comment:
The result of this should probably be cached too.

Reply:
I really don't think this one is necessary.